src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/QuestionSteps.java

```diff
@@ -81,7 +81,12 @@ public void setup() throws Exception {
     @After
     public void cleanUp() {
         for (UUID questionId : questionIds) {
-            questionRepository.deleteById(questionId);
+            try {
+                questionRepository.deleteById(questionId);
+            } catch (Exception e) {
+                // Don't care
+            }
+
```

Comment:
I'll fix this in another branch so that deleted questions are removed from the list